### PR TITLE
Add jingxu97 to owner file

### PR DESCRIPTION
**Pull request by jingxu97**
Add jingxu97 to owner file

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -6,10 +6,12 @@ reviewers:
   - saad-ali
   - saikat-royc
   - verult
+  - jingxu97
```
Comment (msau42) [inline OWNERS:9]: Can you put this in alphabetical order?

Comment (jingxu97) [reply OWNERS:9]: updated. PTAL

File: OWNERS [hunk 2]
```diff
@@ -6,10 +6,12 @@ reviewers:
...
 approvers:
   - davidz627
   - mattcary
   - msau42
   - saad-ali
   - saikat-royc
   - verult
+  - jingxu97
```
Comment (mattcary) [inline OWNERS:17]: both in alphabetical order? :-)

Comment (jingxu97) [reply OWNERS]: somehow I didn't push my changes. Now it is updated. Thanks!

**Pull request merged**